### PR TITLE
Add verbose override fields

### PR DESCRIPTION
**Pull request by jwolski**
@Raynos

### CODE REVIEW

File: error.js [hunk 1]
```diff
@@ -66,9 +67,16 @@ function writeError(req, res, opts, callback) {
     }
 
     if (opts.verbose) {
```
Comment (Raynos) [inline error.js]: We want to make a breaking change and ensure that by default, when not configured, `stack` is not serialized.

Comment (jwolski) [inline error.js]: i see. this seemed more legit than a breaking change. but ill do it

**Comment by Raynos** [comment]
lgtm. #shipit.

**Comment by Raynos** [comment]
- Added you as collaborator
 - Added you as npm owner
 - Published v6.0.0

**Pull request merged**